networks/cardano/tokens/51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "",
+    "decimals": 6,
+    "mldId": "cardano-STUFF-51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b",
+    "orgName": "Stuff.io",
+    "orgUrl": "https://stuff.io",
+    "tokenAddress": "51a5e236c4de3af2b8020442e2a26f454fda3b04cb621c1294a0ef34424f4f4b",
+    "tokenColorTheme": 2,
+    "tokenIcon": "/static/tokens/STUFF.svg",
+    "tokenName": "BOOK",
```

Comment:
wrong token name?

Reply:
This one is a tricky one. This project was changing from `Book,io` -> `Stuff.io`. `Asset Name` is still set to `Book`:

![image](https://github.com/user-attachments/assets/a0d1c9ee-7e68-42e6-bc0e-28c9050e46f4)

Do you think we should keep the `Book` or still swap for `Stuff`?

Reply:
I am now checking with the `Stuff.io` team to see if they will update `BOOK` -> `STUFF`.